kube/kube.yaml

```diff
@@ -24,7 +24,7 @@ spec:
   rules:
   - http:
       paths:
-      - path: /contacts
+      - path: /atlas-contacts-app
```

Comment:
This caused that app was not running in K8s

Reply:
Then we need to add/change examples in readme file

Reply:
readme update